openedx/core/djangoapps/notifications/base_notification.py

```diff
@@ -196,13 +196,15 @@
         'name': 'ora_staff_notification',
         'is_core': False,
         'info': '',
-        'web': False,
+        'web': True,
```

Comment:
Changing defaults would not work on existing preferences , we should do this on a separate PR .
It would require a version change and a name change for this preference type.

Reply:
sure, Let me revert this change